### ISSUE TITLE
Select: placeholder and value

### ISSUE THREAD

**Issue by jfily** [jamesmfriedman/rmwc#533]
- What RMWC Version are you using [major.minor.patch]:

Latest : 5.7.1

- Name your build system [Webpack, Rollup...]:

Webpack

- Describe the bug with as much detail as possible:

I try to make an optional select with the placeholder prop but when I use the "value" prop, the placeholder disapear : https://codesandbox.io/s/rmwc-sandbox-zue4h

- What happened, and what was supposed to happen:

After checking the code, this is intended but I wonder why.
```javascript
{!this.props.value && !this.props.defaultValue && (
   <option value="" disabled={placeholder === ''}>
       {placeholder}
   </option>
)}
```

So is there another way to create an optional select with a placeholder ?

**Comment by jamesmfriedman** [comment]
Thats definitely a bug, and some very shoddy logic! A workaround for now.

https://codesandbox.io/s/rmwc-sandbox-ijrqo

**Comment by jfily** [comment]
This is what i'd do as a fix :

```typescript
{placeholder !== undefined && (
    <option value="" disabled={rest.required}>
        {placeholder}
    </option>
)}
```

**Comment by jamesmfriedman** [comment]
Feel free to PR it If you’d like. Working hard on V6, will definitely get this in there.

**Comment by jamesmfriedman** [comment]
Released and fixed in 5.7.2
